.github/workflows/mingw.yml

```diff
@@ -70,32 +71,30 @@ jobs:
         with:
           ruby-version: ${{ matrix.baseruby }}
 
-      - name: where check
+      - name: Misc system & package info
+        working-directory:
         run: |
           # show where
-          mv /c/Windows/System32/libcrypto-1_1-x64.dll /c/Windows/System32/libcrypto-1_1-x64.dll_
-          mv /c/Windows/System32/libssl-1_1-x64.dll    /c/Windows/System32/libssl-1_1-x64.dll_
           result=true
           for e in gcc.exe ragel.exe make.exe libcrypto-1_1-x64.dll libssl-1_1-x64.dll; do
             echo ::group::$'\033[93m'$e$'\033[m'
             where $e || result=false
             echo ::endgroup::
           done
-          $result
-        working-directory:
-
-      - name: version check
-        run: |
+          mv /c/Windows/System32/libcrypto-1_1-x64.dll /c/Windows/System32/libcrypto-1_1-x64.dll_
+          mv /c/Windows/System32/libssl-1_1-x64.dll    /c/Windows/System32/libssl-1_1-x64.dll_
```

Comment:
Are these lines needed even after upgrading baseruby to 3.2?

Reply:
Removed, and replaced OpenSSL dll's with the new file names in the code above.